consoles/video_stream.pm

```diff
@@ -0,0 +1,395 @@
+# Copyright © 2021 Marek Marczykowski-Górecki
+# Copyright © 2021 SUSE LLC
```

Comment:
```suggestion
# Copyright 2021 Marek Marczykowski-Górecki
# Copyright 2021 SUSE LLC
```

as in https://github.com/os-autoinst/os-autoinst/blob/master/consoles/console.pm#L2